base/scopedvalues.jl

```diff
@@ -40,17 +88,21 @@ julia> sval[]
     Scoped values were introduced in Julia 1.11. In Julia 1.8+ a compatible
     implementation is available from the package ScopedValues.jl.
 """
-mutable struct ScopedValue{T}
-    # NOTE this struct must be defined as mutable one since it's used as a key of
-    #      `ScopeStorage` dictionary and thus needs object identity
-    const has_default::Bool # this field is necessary since isbitstype `default` field may be initialized with undefined value
+mutable struct ScopedValue{T} <: AbstractScopedValue{T}
```

Comment:
```suggestion
mutable struct ScopedValue{T} <: AbstractScopedValue{T}
    # NOTE this struct must be defined as mutable one since it's used as a key of
    #      `ScopeStorage` dictionary and thus needs object identity
```